Tools/Scripts/webkitpy/browserperfdash/browserperfdash_runner.py

```diff
@@ -243,7 +245,17 @@ def run(self):
                 # Run test and save test info
                 with tempfile.NamedTemporaryFile() as temp_result_file:
                     benchmark_runner_class = benchmark_runner_subclasses[self._args.driver]
-                    runner = benchmark_runner_class(plan, self._args.localCopy, self._args.countOverride, self._args.timeoutOverride, self._args.buildDir, temp_result_file.name, self._args.platform, self._args.browser, None)
+                    runner = benchmark_runner_class(plan,
+                                                    self._args.localCopy,
+                                                    self._args.countOverride,
+                                                    self._args.timeoutOverride,
+                                                    self._args.buildDir,
+                                                    temp_result_file.name,
+                                                    self._args.platform,
+                                                    self._args.browser,
+                                                    None, None, None, None,
+                                                    None, None, None, None,
+                                                    self._args.browser_args)
```

Comment:
Instead of passing all those `None` parameters, maybe this can be simplified by simply passing `browser_args=self._args.browser_args` ?